.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+
+jobs:
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        flags:
+          - ""
+          - "--features no_std"
+          - "--features serde"
+          - "--features plugins"
+          - "--features unchecked"
```

Comment:
OK, let's skip `unchecked`, `sync`, `no_optimize`, `only_i64` as they are less likely to have issues...

Reply:
`sync` is quite an involved feature, and it doesn't hurt to include them. It's more of an issue if we need to build a large collection of features for **x** platforms **x** toolchains.

But just adding three is not a big deal!

Reply:
Ah OK!